docs/sidebar.yml

```diff
@@ -53,6 +53,8 @@ sidebar:
               url: docs/reference/implicit-by-name-parameters.html
             - title: Auto Parameter Tupling
               url: docs/reference/auto-parameter-tupling.html
+            - title: Auto Parameter Tupling - More Details
```

Comment:
Following Martin's [comment](https://github.com/lampepfl/dotty/pull/5217#issuecomment-429864003), this should be removed